### PR TITLE
[SWA-128][FEAT] - Update social links, links to Range Liquidity and new Popup

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-128](https://linear.app/swaprhq/issue/SWA-128/update-social-links-and-showcase-new-amm-concentrated-liq)

# Description

* Update Discords links
* Remove $SWPR ClaimModal component
* Add new Range Liquidity header link
* Add new Range Liquidity popup component

# Visual evidence
image

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) The new popup and header item should be visible, and they should redirect the user to `https://swapr.liquidity.eth.limo/`, all Discord links should send a correct invitation to Swapr Discord and the Claim modal to claim $SWPR should be unavailable

### CODE REVIEW

File: public/locales/de/common.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
   "vote": "Abstimmung",
   "charts": "Diagramme",
   "liquidity": "Liquidität",
+  "rangeLiquidity": "Bereich Liquidität",
```
Comment (Diogomartf) [inline public/locales/de/common.json:91]: let's change all of this instances to concentratedLiquidity as it is the term that we use more.

```suggestion
  "concentratedLiquidity": "Bereich Liquidität",
```

Comment (ElRodrigote) [reply public/locales/de/common.json:91]: As we talked offline, we'll move forward with `Liquidity V3` instead Range/Concentrated Liquidity versions. Also moving the `NEW` badge from `DCA` to `Liquidity V3` in our app header

File: src/constants/urls.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const DISCORD_INVITE_LINK = 'https://discord.com/invite/QFkNsjTkzD'
+export const LIQUIDITY_V3_LINK = 'https://swapr.liquidity.eth.limo/'
```
Comment (Diogomartf) [inline src/constants/urls.ts:2]: This link goes to swapbox and we want it to go to the pools and TVL metrics I would say ( `/info/pools` )

```suggestion
export const LIQUIDITY_V3_LINK = 'https://swapr.liquidity.eth.limo/#/info/pools'
```

**Comment by Diogomartf** [comment]
Let's change the banner text to to be consistent:
 
"Provide concentrated liquidity in Gnosis Chain"

I think the most clear term is concentrated liquidity.

**Comment by Diogomartf** [comment]
Also the font is super big, I would decrease it a bit.

image

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -312,13 +296,17 @@ function Header() {
             {t('liquidity')}
             {networkWithoutSWPR && <HeaderLinkBadge label="NOT&nbsp;AVAILABLE" />}
           </HeaderLink>
+          <HeaderLink data-testid="liquidity-v3-nav-link" id="liquidity-v3-nav-link" href={LIQUIDITY_V3_LINK}>
+            {t('liquidityV3')}
+            <NewBadge>NEW</NewBadge>
+            {networkWithoutSWPR && <HeaderLinkBadge label="NOT&nbsp;AVAILABLE" />}
```
Comment (berteotti) [inline src/components/Header/index.tsx:302]: I think `networkWithoutSWPR ` does not make sense here. We are not showing "Not available" on Ethereum. Also, I think we shouldn't show "Not available" on a new product, we want them to click.

File: src/components/Header/index.tsx [hunk 4]
```diff
@@ -279,14 +271,6 @@ function Header() {
...
   return (
     <HeaderFrame>
-      <ClaimModal
-        onDismiss={toggleClaimPopup}
-        newSwprBalance={newSwprBalance}
-        stakedAmount={stakedTokenAmount?.toFixed(3)}
-        singleSidedCampaignLink={
-          data && !loading ? `/rewards/single-sided-campaign/${data.stakeToken.address}/${data.address}` : undefined
-        }
-      />
```
Comment (berteotti) [inline src/components/Header/index.tsx:289]: Why are we removing this?

File: src/components/Popups/LiquidityV3Popup.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+
+import GnoTokenLogoDark from '../../assets/images/gnosis-chain-logo-dark.svg'
+import SwaprTokenLogoDarkWithRing from '../../assets/images/swapr-logo-dark-with-ring.svg'
+import { LIQUIDITY_V3_LINK } from '../../constants'
+
+export const LiquidityV3Popup = () => {
+  return (
+    <FlexContainer onClick={() => document.location.assign(LIQUIDITY_V3_LINK)}>
```
Comment (berteotti) [inline src/components/Popups/LiquidityV3Popup.tsx:9]: Can we define `FlexContainer` as a anchor element `<a>` and use `href` and `target` props to open the link? This should open on a new window, so user flow doesn't get disrupted by clicking a link.

File: src/constants/urls.ts (hunk 2 above)
Comment (berteotti) [inline src/constants/urls.ts:2]: Can we define this as env variables with fallback?

File: src/state/application/hooks.tsx [hunk 6]
```diff
@@ -111,6 +112,19 @@ export function useNotificationPopup() {
...
+    toast.info(<LiquidityV3Popup />, {
+      autoClose: false,
+      icon: false,
+      position: toast.POSITION.BOTTOM_RIGHT,
+      progressStyle: {
+        background: 'hsla(0,0%,100%,.7)',
+      },
+    })
+  }, [])
+}
```
Comment (berteotti) [inline src/state/application/hooks.tsx:126]: I think we can get rid of `useCallback` in this case. This function will only be declared once, so the use of `useCallback` becomes redundant here

**Review by berteotti — COMMENTED** [review]
Code looks good, made a few comments to improve little things and some design related stuff.

File: src/components/Header/index.tsx (hunk 4 above)
Comment (ElRodrigote) [reply src/components/Header/index.tsx:289]: This was a request from @0xVenky as far as I understood in the ticket:
https://linear.app/swaprhq/issue/SWA-128/update-social-links-and-showcase-new-amm-concentrated-liq

image

File: src/components/Popups/LiquidityV3Popup.tsx (hunk 5 above)
Comment (ElRodrigote) [reply src/components/Popups/LiquidityV3Popup.tsx:10]: Agree, will do!

File: src/components/Header/index.tsx (hunk 3 above)
Comment (ElRodrigote) [reply src/components/Header/index.tsx:302]: Ok! Removing it now!

File: src/constants/urls.ts (hunk 2 above)
Comment (ElRodrigote) [reply src/constants/urls.ts]: Good idea, will do it now!

File: src/state/application/hooks.tsx (hunk 6 above)
Comment (ElRodrigote) [reply src/state/application/hooks.tsx:126]: I just tried this out and it works, pushing the updates in a bit!

**Comment by berteotti** [comment]
Screenshot 2023-12-22 at 10 20 54

Can we have a smaller font on mobile?

File: src/components/Popups/LiquidityV3Popup.tsx (hunk 5 above)
Comment (Diogomartf) [reply src/components/Popups/LiquidityV3Popup.tsx]: `target="_blank"` to open on new tab

**Review by Diogomartf — APPROVED** [review]

File: src/constants/urls.ts [hunk 7]
```diff
@@ -0,0 +1,2 @@
+export const DISCORD_INVITE_LINK = process.env.DISCORD_INVITE_URL ?? 'https://discord.com/invite/QFkNsjTkzD'
+export const LIQUIDITY_V3_LINK = process.env.LIQUIDITY_V3_URL ?? 'https://swapr.liquidity.eth.limo/#/info/pools'
```
Comment (berteotti) [inline src/constants/urls.ts:2]: 🔥

File: src/pages/Pools/LiquidityV3Banner/index.tsx [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <FlexLinkContainer href={LIQUIDITY_V3_LINK} target="_blank">
+      <LiquidityPairLogo
+        leftLogoSrc={SwaprTokenLogoDarkWithRing}
+        leftLogoWidth={56}
+        height={56}
+        rightLogoSrc={GnoTokenLogoDark}
+        rightLogoWidth={24}
+      />
+      <LiquidityBannerText>Pick your range when providing liquidity, only in Gnosis chain</LiquidityBannerText>
```
Comment (berteotti) [inline src/pages/Pools/LiquidityV3Banner/index.tsx:40]: Just a small typo.

```suggestion
      <LiquidityBannerText>Pick your range when providing liquidity, only in Gnosis Chain</LiquidityBannerText>
```

**Review by berteotti — APPROVED** [review]
Good Job Rorry, just a small typo on the banner. It can be merged after that fix.

**Pull request merged**